### DISCUSSION TITLE
Streak system similar to Duolingo

### DISCUSSION THREAD

**Discussion by guy23456** [monkeytypegame/monkeytype#4847] [Ideas] [answered]
I have an idea for a feature that could help people improve their keyboarding speed; a streak system similar to Duolingo's may help. I understand if there are reasons to avoid adding this feature, but I feel that it would be nice to have.

**Comment by sleeepstorm** [comment, marked as answer]
There is a streak system implemented into monkeytype. If you are logged in and viewing your profile, you can see your current streak.
